cli/running/process_controller.go

```diff
@@ -111,3 +111,38 @@ func (pc *processController) Stop(waitTimeout time.Duration) error {
 		return err
 	}
 }
+
+// Quit terminates the process.
+func (pc *processController) Quit(waitTimeout time.Duration) error {
+	if !pc.IsAlive() {
+		return nil
```

Comment:
Process is not terminated by sigquit, we need to write about it, because a user will search for a core dump.